drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_services_loader.rb

```diff
@@ -427,7 +433,7 @@ def row_enrollment_id(row)
           },
         },
         'Individual TBSS' => {
-          service_type: 'Individual TBSS',
+          service_type: 'TBSS',
```

Comment:
@ttoomey I pushed a small change to 2 service type names per [request](https://www.pivotaltracker.com/n/projects/2591838/stories/187215113) https://github.com/greenriver/hmis-warehouse/pull/4101/commits/d9d8984b4c1c31ddd4ae3782e12d2c6f385fb7fc